src/routers/alpha-router/alpha-router.ts

```diff
@@ -1645,6 +1652,102 @@ export class AlphaRouter
       }
     }
 
+    let newSetCachedRoutesPath = false;
+    const shouldEnableCachedRoutesCacheInvalidationFix =
+      Math.random() * 100 <
+      (this.cachedRoutesCacheInvalidationFixRolloutPercentage ?? 0);
```

Comment:
eventually this will roll out to 100% of cache entries even with a small percent enabled, if I understand correctly

Reply:
yes